### ISSUE TITLE
Firedrake-Install problems on fresh Ubuntu 16.04

### ISSUE THREAD

**Issue by da1910** [firedrakeproject/firedrake#1044]
I've been trying all morning to get firedrake installed on my ubuntu laptop using the install script, and it seems to be failing during the build of PETSc when configuring EIGEN, I've included the install log below. I subsequently tried this on a clean install of Ubuntu 16.04 and the problem recurs.

[firedrake-install.log.txt](https://github.com/firedrakeproject/firedrake/files/912972/firedrake-install.log.txt)

**Comment by dham** [comment]
Hi Doug,

This is an upstream issue with PETSc: they've managed to break the Eigen install. Eigen is only used for hybridisation solvers and adaptivity. Assuming you don't need those, you can work around this by disabling the Eigen download:

```
PETSC_CONFIGURE_OPTIONS="--download-eigen=0" python firedrake-install
```

We hope that this issue will go away in a few days when PETSc push a fix. 

A bulletin on this issue went out on the Firedrake mailing list yesterday. We recommend that all Firedrake users subscribe to the list exactly to receive this sort of update. See: http://firedrakeproject.org/contact.html

**Comment by miklos1** [comment]
Fixed by #1045, I hope.
